src/System.Text.Json/tests/Configurations.props

```diff
@@ -4,6 +4,9 @@
     <BuildConfigurations>
       netcoreapp;
       uap-Windows_NT;
+      <!-- Only included for testing and validating that the sources are netstandard compatible. -->
+      <!-- This is meant to help with producing a source package and not to ship a netstandard compatible binary. -->
+      netstandard;
```

Comment:
NIT: This could also be netfx as it will be the only vertical where we would test the netstandard facade. That said, having it being netstandard I believe is better, that way we can use this as part of our netstandard test suite in the future.